src/main/webapp/scripts/session-script.test.js

```diff
@@ -63,6 +63,28 @@ test('tests copy and paste', () => {
 });
 
 test('addOnClickTo', () => {
+  addOnClickToPageBuilder();
+  const sessionInfoSpan = document.getElementById('session-info-span');
+  const sessionInfoDiv = document.getElementById('session-info-div');
+  const close = document.getElementsByClassName('close').item(0);
+  const sessionIdInput =
+      document.getElementsByClassName('session-id-input').item(0);
+  sessionscript.addOnClickToElements();
```

Comment:
This function name confused me for literal minutes. Rename this to addOnClickListenerToElements(). The listener part makes it more udnerstandable.